lib/index.js

```diff
@@ -273,6 +273,9 @@ function makeEventError(error) {
 function extractErrors(obj) {
   var errors = [];
 
+  if (obj instanceof Error) {
+    errors.push(obj);
```

Comment:
Do we still want to try to find nested errors after this, or should we just return `[obj]` here? I'm ok with either way, just bringing up the question